src/core/workload.py

```diff
@@ -8,17 +8,24 @@
 import string
 from abc import ABC, abstractmethod
 
-from literals import PATHS
+from literals import PATHS, Role
 
 
-class KafkaPaths:
+class CharmedKafkaPaths:
```

Comment:
**question:** Any reason for the name-change?

Reply:
I just got confused at some point when I accessed a cruise control path using something called `KafkaPath`. This is a ME problem, I can revert this change if you'd like.

Reply:
No, I like that reasoning! Makes sense 👍🏾